### PR TITLE
feat: polishing cli arguments and help messages

### PR DESCRIPTION
**Pull request by ricardomaraschini**
polishing cli arguments and help messages

### CODE REVIEW

**Review by laverya — DISMISSED** [review]

File: cmd/embedded-cluster/install.go [hunk 1]
```diff
@@ -265,7 +265,7 @@ func runOutro(c *cli.Context) error {
 // file is created. Resulting kubeconfig is stored in the configuration dir.
 var installCommand = &cli.Command{
 	Name:  "install",
-	Usage: "Installs and starts a new cluster",
+	Usage: "Install a cluster",
```
Comment (ajp-io) [inline cmd/embedded-cluster/install.go:268]: This is more than a cluster to the end user. In fact, they think of it as the app more so than the cluster. WDYT about "Install \<binary name>"? Maybe "Install \<binary name> cluster", but in a single-node situation, the word cluster feels confusing.

This lines up with the `Install and manage %s", name` wording you used below too.

File: cmd/embedded-cluster/install.go [hunk 2]
```diff
@@ -280,7 +280,7 @@ var installCommand = &cli.Command{
 		},
 		&cli.StringSliceFlag{
 			Name:  "disable-addon",
-			Usage: "Disable addon during install/upgrade",
+			Usage: "Disable addon during install",
```
Comment (ajp-io) [inline cmd/embedded-cluster/install.go:283]: But I'm all for just getting rid of this one
```suggestionx
			Usage: "Disable add-on during install",
```

File: cmd/embedded-cluster/reset.go [hunk 3]
```diff
@@ -299,7 +299,7 @@ var resetCommand = &cli.Command{
 		},
 		&cli.BoolFlag{
 			Name:  "force",
-			Usage: "Ignore errors encountered when resetting the node. Implies --no-prompt.",
+			Usage: "Ignore errors encountered when resetting the node. Implies --no-prompt",
```
Comment (ajp-io) [inline cmd/embedded-cluster/reset.go:302]: If we're going to remove periods, let's do something like this
```suggestion
			Usage: "Ignore errors encountered when resetting the node (implies --no-prompt)",
```

**Review by ajp-io — COMMENTED** [review]
I wonder if we need the `help` command if we have `--help, -h` already. Nbd, just thinking out loud.

File: cmd/embedded-cluster/install.go (hunk 2 above)
Comment (ricardomaraschini) [reply cmd/embedded-cluster/install.go:283]: This will be done in a separate PR.

File: cmd/embedded-cluster/install.go (hunk 1 above)
Comment (ricardomaraschini) [reply cmd/embedded-cluster/install.go:268]: OK, `Install <binary name>` it is.

Comment (ricardomaraschini) [reply cmd/embedded-cluster/install.go:268]:
```suggestion
	Usage: fmt.Sprintf("Install %s", defaults.BinaryName()),
```

**Review by laverya — APPROVED** [review]

**Pull request merged**